app/jobs/transfer_client_job.rb

```diff
@@ -0,0 +1,24 @@
+class TransferClientJob < ActiveJob::Base
+  queue_as :lupo_background
+
+  def perform(symbol, options = {})
+    client = Client.where(symbol: symbol).first
+
+    if client.present? && options[:target_id].present?
+      options = {
+        from_id: Doi.minimum(:id).to_i,
+        until_id: Doi.maximum(:id).to_i,
+        filter: { client_id: symbol.downcase },
+        label: "[ClientTransfer]",
+        job_name: "UpdateProviderIdJob",
+        target_id: options[:target_id],
+      }
+
+      Doi.loop_through_dois(options)
+
+      Rails.logger.info "[Transfer] DOIs transfer has started for #{client.symbol} to #{options[:target_id]}."
```

Comment:
Are you transferring both repositories and DOIs?

Reply:
all dois need to be "updated" because provider and provider_id are in the index https://github.com/datacite/lupo/blob/4cb634d/app/models/doi.rb#L201 . 

Maybe we can adjust to language. "Assign" or "update" would be better ??

Reply:
I like `update`.